css/style.scss

```
@@ -101,13 +110,13 @@ Projects
 Contact Form
 ************************/
 form {
-  color: #FFF;
+  color: $textcolor2;
 }
 
```

The input tags can be nested such as 
```suggestion
 input {
  &[type=text] {
   
   }
   
   &[type=email] {
   
   }
 }
```

You get the idea. 

The & in scss refers to the parent object, in this case it would be the input.